app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1581,6 +1580,94 @@ private function resetImport($filepath)
         unlink($filepath);
     }
 
+    public function exportCurrentListAction(){
```

Comment:
I'm not sure what you mean by the *existing* export functionality. Is there already a standard way to export things in Mautic? I've found two export functions but they seem to be separate.
Are you saying I should try to refactor them along with this lead export functionality?